Tests/SentryTests/Performance/SentryTracerTests.swift

```diff
@@ -870,34 +892,43 @@ class SentryTracerTests: XCTestCase {
         assertSpan("Initial Frame Render", appStartMeasurement.didFinishLaunchingTimestamp, fixture.appStartEnd)
     }
 
+    private func assertAppStartMeasurementOn(transaction: Transaction, appStartMeasurement: SentryAppStartMeasurement) {
+        let serializedTransaction = transaction.serialize()
+        let measurements = serializedTransaction["measurements"] as? [String: [String: Int]]
+
+        XCTAssertEqual(["app_start_warm": ["value": 500]], measurements)
```

Comment:
If this `500` is meant to test exceeding the maximum launch duration, we could convert that macro to a constant and expose it to this test.

Reply:
It depends on `fixture.appStartDuration`. Good point, I changed it.